boards/phytec/reel_board/reel_board.dts

```diff
@@ -70,12 +70,10 @@
 				border-waveform = <0x71>;
 				dummy-line = <0x1a>;
 				gate-line-width = <0x08>;
-				lut = [
-					22 55 AA 55 AA 55 AA 11
-					00 00 00 00 00 00 00 00
-					1E 1E 1E 1E 1E 1E 1E 1E
-					01 00 00 00 00
-				];
+				lut = [22 55 AA 55 AA 55 AA 11
+				       00 00 00 00 00 00 00 00
+				       1E 1E 1E 1E 1E 1E 1E 1E
+				       01 00 00 00 00];
```

Comment:
That does not look right either. This followed the usual C-style initialization that we have in the tree.

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/93123